### DISCUSSION TITLE
Disable User Account as Admin

### DISCUSSION THREAD

**Discussion by jwheeler31** [stoatchat/discussions#770] [Q&A]
According to the [disable account](https://developers.revolt.chat/developers/api/reference.html?search=#tag/account/POST/auth/account/disable) endpoint, it looks like only the user themselves can disable the account. Is this correct? Is it possible to perform the disable as an admin from an external source, or would it require modifying the database directly?

**Comment by IAmTomahawkx** [comment]
Hi, this would indeed be a direct database edit. This is currently how all our platform tools operate.
